spec/controllers/control_panel/control_panel_spec.rb

```diff
@@ -1,20 +1,174 @@
 require 'rails_helper'
 
-RSpec.describe ControlPanel::StaticPagesController do
+RSpec.describe ControlPanel::BoardsController do
   let(:user) { create :user }
-  describe "get index" do
-    it "don't let you pass if you are not logged in"  do
-      get :index
-      expect(response).to redirect_to(root_path)
+  let(:board) {create :board, owner_id: user.id}
+  describe "index" do
+
+    context "before sign in" do
+      it "don't let you pass if you are not logged in"  do
+        get :index
+        expect(response).to redirect_to(root_path)
+      end
+      it "flashes user auth warning message" do
+        get :index
+        expect(flash[:alert]).to eq I18n.t('user.auth.failure')
+      end
     end
+
     context "after sign in" do
-      before do
-        sign_in(user)
-      end
+      before { sign_in(user) }
+
       it "let you pass if you are logged in" do
         get :index
-        expect(response).to render_template(:index)
+        expect(response).to render_template :index
         end
     end
   end
+
+  describe "new" do
+    it "go into right template" do
```

Comment:
don't forget about grammar :P 
`it 'goes...`
and think about what this test expects, so I think it should be ` it 'renders template new'`